dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -52,9 +55,10 @@ export default defineComponent({
     useMobileView() {
       return this.injectedUseMobileView;
     },
-    isCompanyIdValid() {
-      const uuidRegexExp = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
-      return uuidRegexExp.test(this.companyId);
+    isClaimPanelVisible() {
+      return (
+        !this.isUserDataOwner && this.authenticated && isCompanyIdValid(this.companyId) && !this.hasCompanyDataOwner
```

Comment:
Warum das !this.hasCompanyDataOwner?
Eine Company kann doch mehr als einen DataOwner haben, so war zumindest mein Verständnis. Dann sollte das Banner doch angezeigt sein, solange die Company valide ist, der Nutzer eingeloggt ist und der Nutzer selber noch nicht Data Owner ist, oder?

Reply:
Nein, sobald es mindestens einen Data Owner gibt, soll das Panel nicht mehr sichtbar sein. Alternativ kann man über das context-menu den ownership claimen.

Reply:
Ok, dann passt das